doc/classes/VisualShaderNodeBooleanConstant.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		Has only one output port and no inputs.
-		Translated to [code skip-lint]bool[/code] in the shader language.
+		Translated to [code skip-lint]bool[/code] in GDShader.
```

Comment:
These all seem good. But the translations will break, so we should take the opportunity to go over them again and make improvements.

Reply:
It may prove difficult to add anything more of substance especially from contributors which lack knowledge on Visual Shaders.

Reply:
I mentioned in another comment that I'm willing to do this (as a followup PR, if it makes more sense). I'm already working with the VisualShaderNode code.